.github/ISSUE_TEMPLATE/prisma-cli-studio-bug-report.yml

```diff
@@ -0,0 +1,53 @@
+name: Prisma CLI `prisma studio` report
+description: Something not working as expected when using `prisma studio` in Prisma CLI
+labels: "kind/bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve Prisma! 🙏 Please follow the sections in the template and provide as much information as possible about your problem.
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: Steps to reproduce the behavior
+      value: |
+        <!--
+        1. Go to '...'
+        2. Change '....'
+        3. Run '....'
+        4. See error
+        -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+  - type: textarea
+    attributes:
+      label: Prisma information
+      description: Your Prisma version (prisma -v or npx prisma -v), Prisma schema
+      value: |
+        <!-- Do not include your database credentials when sharing your Prisma schema! -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment & setup
```

Comment:
I am wondering how relevant the Environment section is here for Studio from the CLI? 🤔 what are your thoughts here @d-ivashchuk

Reply:
I think it could be relevant as we are executing an iframe in some environment 🤔 so it may make a difference if user is running the pdp on safari or chorme and has data browser failing

Reply:
yeah, you got it right!